product_docs/docs/pem/8/pem_online_help/04_toc_pem_features/09_pem_alerting/04_pem_custom_alert_templates.mdx

```diff
@@ -104,6 +104,39 @@ After defining a new alert template, click the `Add/Change` button to save the d
 
 After defining a template, you can use the [Manage Alerts](01_pem_alerting_dialog/#pem_alerting_dialog) tab to create and enable an alert based on the template.
 
+## Exporting or Importing an Alert Template
+
+From PEM 8.2 onwards, you can export or import the alert templates to another PEM Server.
+
+To `Export` the Alert Template, select any alert template from the `Alert Templates` tab and then select the `Export` icon in the upper-right corner of the table. Select `Save File` option and then select `ok`, it will generate the `JSON` file.
+
+<figure><img src="../../images/alert_template_export.png" class="align-center" alt="Alert Template - Export" /><figcaption aria-hidden="true"><em>Alert Template - Export</em></figcaption></figure>
+
+To `Import` the Alert Template, go to the `Alert Templates` tab and then select the `Import` icon in the upper-right corner.
+
+<figure><img src="../../images/alert_template_import.png" class="align-center" alt="Alert Template - Import" /><figcaption aria-hidden="true"><em>Alert Template - Import</em></figcaption></figure>
+
+Click on the `Browse` button to select the `JSON` file with the code to be imported and then click `Import`.
+
+<figure><img src="../../images/alert_template_import_browse.png" class="align-center" alt="Alert Template - Import Browse" /><figcaption aria-hidden="true"><em>Alert Template - Import Browse</em></figcaption></figure>
+
+After selecting the file to import, you can select the following checkboxes:
+
+-   `skip exiting` - If you select this checkbox then it will skip the alert template if it already exists.
+-   `skip existing dependent probe` - The alert templates are dependent on probes. If you select this checkbox then it will skip the dependent probe if it already exists.
+
+If both the checkboxes are selected and the alert template already exists, then it skips importing the alert template with below message:
+
+<figure><img src="../../images/alert_template_skip_both.png" class="align-center" alt="Alert Template - Skip Message" /><figcaption aria-hidden="true"><em>Alert Template - Skip Message</em></figcaption></figure>
```

Comment:
The syntax for images need to be replaced with ! [Alternate_text](<path_to_image_file>/<image_filename.png>)

Reply:
There were several hundred of these already in the PEM online help; I've rewritten all of them.